src/testPane.ts

```diff
@@ -1273,8 +1273,8 @@ export function updateCodedTestCases (editor:any) {
         // note: the same file should never be associated with more than one unit
         let commandToRun: string = 
           `${clicastCommandToUse} ${getClicastArgsFromTestNode(testNode)} test coded refresh`;
+        vectorMessage (`Refreshing coded test file: ${filePath} for environment: ${enviroPath}`);
```

Comment:
This PR changes `vectorMesage` to be async. We have many places in the project where we call `vectorMessage` inside a **non-async** function. This means that the function execution will continue right away and will not wait for message display to be done. We cannot "await" the `vectorMessage` inside a non-async functions, so the message can not be displayed before a long running process. 

Is this a desired behavior? If we need an async variant of `vectorMessage` for use in particular **async functions**, we could implement it separately, while still keeping the synchronous variant we had before

Reply:
I'm pretty sure that the single function will operate this way ... if called with await (from an aysnc function) the message will be forced to be displayed and otherwise it will work as it did before.  This gives us the option to make the caller async in cases where we want the message displayed, and that's what I did for the two lambda functions.

Reply:
ok, all good then